README.md

```diff
@@ -59,6 +59,13 @@ registerIconLibrary('system', {
 });
 ```
 
+5. Valgfritt: Vue 3, Eslint-regler og VS Code
+   Hvis du bruker VS Code og starter et nytt prosjekt gjennom npm run create vue@latest og bruker standard eslint-reglene, kan du
+   få feil fra eslint-regler og automatisk retting fra VS-kode. Du kan legge til disse ESLint-reglene før for å unngå disse feilene:
```

Comment:
Nesten perfekt norsk. Bytt ut 'før' med 'for', så er vi der.

Reply:
Toppen, takk for rættningen! Ska førsøka førbættra min norska. :D